examples/discover/main.go

```diff
@@ -59,6 +61,9 @@ func main() {
 	srvcs, err := device.DiscoverServices(nil)
 	must("discover services", err)
 
+	// buffer to retrieve characteristic data
+	buf := make([]byte, 255)
```

Comment:
Note that you can use a smaller buffer here, `copy` will simply use the smallest of the two slices (so if the sent value is longer than `buf` it'll simply get cut off).